proto/tribune_tts.proto

```diff
@@ -48,13 +60,70 @@ message SynthesizeRequest
 // Provides information to the synthesizer that specifies how to process the request.
 message SynthesizeConfig
 {
-    // Desired sampling frequency in hertz of synthesized audio.
+    // Desired sampling frequency in hertz of synthesized audio. The value 0 means use the default Synthesizer sampling rate.
     int32 sample_rate_hertz = 1;
+
+    // Requested format of the output audio stream.
+    OutputFormat output_format = 2;
+
+    // ISO 639-1 code of the language of text to be synthesized (may be overriden by SSML tags in request text).
+    string language = 3;
+
+    // Optional. Overrides the default synthesized speech parameters
+    AudioConfig audio_config = 4;
+
+    // Requested voice to be used to synthesize the text (may be overriden by SSML tags in request text).
+    Voice voice = 5;
+}
+
+// The requested format of the response audio data.
+enum OutputFormat
+{
+    PCM16 = 0;       // Uncompressed 16-bit signed integer samples.
```

Comment:
PCM audio is returned with- or without- the header?

Reply:
Without the header. Clarified in comment.